### PR TITLE
nsqadmin_to_slack: send nsqadmin event notifications to Slack

### PR DESCRIPTION
**Pull request by jehiah**
This is a tool extracted out of the Bitly repo to send nsqadmin notifications to a Slack channel. Debatable if this should land in the main repo, but ¯\_(ツ)_/¯.

### CODE REVIEW

**Comment by ploxiln** [comment]
Just throwing out an alternative idea: a new repo in the nsqio organization.

**Comment by ploxiln** [comment]
Another possibly crazy idea, inspired by but not related to this PR: make nsq_pubsub a separate repo before the 1.0 release

**Comment by mreiferson** [comment]
Good questions — the `nsqio` org does afford us the opportunity to break some stuff out (and `nsqadmin_to_slack` utility is particular to one service provider).

re: `nsq_pubsub`, my preference would be removal because I think it's broken (if I remember correctly?) and hasn't been touched in years. It's not an example we should encourage.

File: apps/nsqadmin_to_slack/nsqadmin_to_slack.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	lookupdHTTPAddrs := app.StringArray{}
+	flag.Var(&nsqdTCPAddrs, "nsqd-tcp-address", "nsqd TCP address (may be given multiple times)")
+	flag.Var(&lookupdHTTPAddrs, "lookupd-http-address", "lookupd HTTP address (may be given multiple times)")
+	cfg := nsq.NewConfig()
+	flag.Var(&nsq.ConfigFlag{cfg}, "consumer-opt", "option to passthrough to nsq.Consumer (may be given multiple times, http://godoc.org/github.com/nsqio/go-nsq#Config)")
+
+	flag.Parse()
+
+	if *showVersion {
+		fmt.Printf("nsqadmin_to_hipchat v%s\n", version.Binary)
```
Comment (jehiah) [inline apps/nsqadmin_to_slack/nsqadmin_to_slack.go:38]: @danielhfrank I just noticed this =)

 your [inspiration](https://github.com/danielhfrank/nsqadmin2hipchat) lives on

Comment (danielhfrank) [reply apps/nsqadmin_to_slack/nsqadmin_to_slack.go:38]: Ha, now that's a trip! Ahead of my time, it seems

**Comment by mreiferson** [comment]
This is going to land in a new repo.

**Comment by jehiah** [comment]
This is now available at https://github.com/nsqio/nsqadmin_to_slack

**Pull request closed without merging**